common/types/replicator_test.go

```diff
@@ -800,3 +800,506 @@ func TestReplicationTask_GetDomainTaskAttributes(t *testing.T) {
 	res = nilStruct.GetDomainTaskAttributes()
 	assert.Nil(t, res)
 }
+
+func TestReplicationTask_GetSyncActivityTaskAttributes(t *testing.T) {
+	syncActivityTask := &SyncActivityTaskAttributes{}
+	testStruct := ReplicationTask{
+		SyncActivityTaskAttributes: syncActivityTask,
+	}
+
+	res := testStruct.GetSyncActivityTaskAttributes()
+	assert.Equal(t, syncActivityTask, res)
+
+	var nilStruct *ReplicationTask
+	res = nilStruct.GetSyncActivityTaskAttributes()
+	assert.Nil(t, res)
```

Comment:
only if you feel like it:

you can probably compact and simplify these with
```go
assert.Zero(t, nilStruct.GetWhatever())
```
since they're all returning zero values, and that'll be type-agnostic.

Reply:
Got it, thanks